README.md

```diff
@@ -289,20 +278,10 @@ Example response:
 ```
 
 ### Deployment
-NOTE: Gubernator uses `etcd`, Kubernetes or round-robin DNS to discover peers and
+NOTE: Gubernator uses `memberlist` Kubernetes or round-robin DNS to discover peers and
```

Comment:
```suggestion
NOTE: Gubernator uses `memberlist`, Kubernetes or round-robin DNS to discover peers and
```

Just to make it clear that memberlist != K8S

Reply:
I would go as far as adding the Oxford comma, too.